relay-server/src/services/project_upstream.rs

```diff
@@ -79,7 +79,10 @@ impl UpstreamQuery for GetProjectStates {
 /// The wrapper struct for the incoming external requests which also keeps addition information.
 #[derive(Debug)]
 struct ProjectStateChannel {
+    // Main broadcast channel.
     channel: BroadcastChannel<ProjectFetchState>,
+    // Additional broadcast channels tracked from merge operations.
+    other: Vec<BroadcastChannel<ProjectFetchState>>,
```

Comment:
nit: Could replace `channel, other` by a single `SmallVec`, to clarify that there is no real difference between the channels.